shippo/api_requestor.py

```diff
@@ -86,26 +85,37 @@ def handle_api_error(self, rbody, rcode, resp):
         else:
             raise error.APIError(rbody, rcode, resp)
 
+    @staticmethod
+    def get_python_version() -> str:
+        return sys.version.split(' ')[0]
+
+    @staticmethod
+    def get_shippo_user_agent_header(configuration: Configuration) -> str:
+        key_value_pairs = []
+        key_value_pairs.append('/'.join([configuration.app_name, configuration.app_version]))
+        key_value_pairs.append('/'.join([configuration.sdk_name, configuration.sdk_version]))
+        key_value_pairs.append('/'.join([configuration.language, APIRequestor.get_python_version()]))
+        return ' '.join(key_value_pairs)
+
     def request_raw(self, method, url, params=None):
         """
         Mechanism for issuing an API call
         """
-        from shippo.config import api_version
-        from shippo.config import app_name
-        from shippo.config import app_version
-
-        # app_name and app_version can be specified in shippo-python-client/shippo/config
-        if app_name == '':
-            app_name = 'PythonApp'
-
-        if app_version == '':
-            app_version = '1.0'
 
         if self.api_key:
             my_api_key = self.api_key
         else:
-            from shippo.config import api_key
-            my_api_key = api_key
+            my_api_key = config.api_key
+
+        if not config.app_name:
```

Comment:
Can we update the ReadMe as well for this and redefine it? Also, this release is not supposed to cause breaking change while instantiating the SDK object, should we keep App name/version optional or have it default to an empty string/default name?

Reply:
Yeah, I was thinking that it can be a bit too restrictive. Defaulting to an empty string is good imho as well.